components/sysutils/bat/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025 Till Wegmueller
+#
+
+BUILD_STYLE = cargo
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		bat
+HUMAN_VERSION =			0.25.0
+COMPONENT_SUMMARY =     A cat(1) clone with wings. 
+COMPONENT_ARCHIVE_HASH =	\
```

Comment:
Does this have a project page?

Reply:
> Does this have a project page?

The cargo build style points by default to crates.io (which is usually sufficient), so there is no need to add `COMPONENT_PROJECT_URL` to component's `Makefile`.  See `make-rules/cargo-defaults.mk` for details.